main.tf

```diff
@@ -66,17 +66,17 @@ resource null_resource "notify_bridgecrew" {
 
   provisioner "local-exec" {
     command = <<CURL
-      curl --request PUT 'https://www.bridgecrew.cloud/api/v1/integrations/csp' \
-      --header 'Authorization: ${var.bridgecrew_token}' \
-      --header 'Content-Type: application/json' \
-      --data-raw '${jsonencode({"customerName": var.org_name,
+      curl --request PUT "https://www.bridgecrew.cloud/api/v1/integrations/csp" \
+      --header "Authorization: ${var.bridgecrew_token}" \
+      --header "Content-Type: application/json" \
+      --data-raw "${jsonencode({"customerName": var.org_name,
                                 "version": local.version,
                                 "credentials": {
                                 "subscriptionId":   data.azurerm_subscription.subscription.subscription_id,
                                 "subscriptionName": data.azurerm_subscription.subscription.display_name,
                                 "tenantId": data.azurerm_subscription.subscription.tenant_id,
                                 "clientId": azuread_application.bridgecrew_app.application_id,
-                                "clientSecret": random_string.password.result}})}'
+                                "clientSecret": random_string.password.result}})}"
```

Comment:
@kartikp10 There's just 1 issue with this - `jsonencode` turns `{"customerName": var.org_name}` to `{"customerName": "some-org"}`. This is valid JSON string: 
```json
'{"customerName": "some-org"}'
```
This, however, is not:
```json
"{"customerName": "some-org"}"
```

Reply:
@nimrodkor Good point. I tested this specific use case in the terraform console and found that the quotes are escaped. 
For example:
```
> jsonencode({"customerName": "some-org"})
"{\"customerName\":\"some-org\"}"
```
I think this should work right?

Reply:
Ah, I think this will still result in `""{\"customerName\":\"some-org\"}""` so this won't work. You're right.

Reply:
Seems like it doesn't work @kartikp10 , tried deploying with this:
```sh
curl: (3) unmatched close brace/bracket in URL position 51
```

Reply:
Yeah, thanks for testing it out. I'm not sure if this is worth fixing for windows.